File: Unidad 3/Java/Programa20U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa20u3;

/**
 *
 * @author Alexander Gonzalez
 */
public class Programa20U3 {

    public static void main(String[] args) {
        int dia = 1,ahorroDiario = 3,ahorroAnual = 0;

        while (dia <= 365) {
            ahorroAnual += ahorroDiario;
            System.out.println("Día " + dia + ": Ahorro diario= " + ahorroDiario);
            ahorroDiario *= 3;

            dia++;
        }

        System.out.println("Ahorro anual: " + ahorroAnual+" pesos ");
    }
}

File: Unidad 3/C#/Programa20U3.cs
// See https://aka.ms/new-console-template for more information
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Programa20U3
    {
        static void Main(string[] args)
        {
                int dia = 1, ahorroDiario = 3, ahorroAnual = 0;

                while (dia <= 365)
                {
                    ahorroAnual += ahorroDiario;
                Console.WriteLine("Día " + dia + ": Ahorro diario= " + ahorroDiario);
                    ahorroDiario *= 3;

                    dia++;
                }

            Console.WriteLine("Ahorro anual: " + ahorroAnual + " pesos ");
      
        }
    }
}